app/AccountancyModule/CampModule/presenters/ParticipantPresenter.php

```diff
@@ -91,6 +91,8 @@ protected function startup() : void
 
     public function renderDefault(int $aid, ?int $uid = null, bool $dp = false, ?string $sort = null, bool $regNums = false) : void
     {
+        $this->setLayout('layout2');
```

Comment:
u campu by mělo jít přehodit už celý layout na nový a tedy to naopak v jednotlivých presenterech odebrat

Reply:
Tohle bych rád udělal až v samostatném PR, který odstraní vše týkající se starého designu